.vscode/settings.json

```diff
@@ -8,5 +8,16 @@
   "eslint.run": "onSave",
   "typescript.preferences.importModuleSpecifier": "non-relative",
   "spellright.language": ["en"],
-  "spellright.documentTypes": ["markdown"]
+  "spellright.documentTypes": ["markdown"],
+  "files.exclude": {
+    "**/.git": true,
+    "**/.svn": true,
+    "**/.hg": true,
+    "**/CVS": true,
+    "**/.DS_Store": true,
+    "**/Thumbs.db": true,
+    "**/key.pem": true,
+    "**/cert.pem": true,
+    "**/node_modules": true
```

Comment:
These made my life easy while scanning through the packages in VSCode side bar.

Reply:
I wouldn't mind these I guess, I always add some and have to keep 'em from being committed myself

Reply:
I'm not sure about this; it's not very common practise to commit these in knowing the .vscode configs of other projects.

Reply:
Let's keep then at the user level configs. These don't belong into the project.